dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.java11_fat;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.MaximumJavaLevel;
+import componenttest.annotation.MinimumJavaLevel;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import componenttest.topology.utils.HttpUtils;
+
+@RunWith(FATRunner.class)
+@MinimumJavaLevel(javaLevel = 16)
+@MaximumJavaLevel(javaLevel = 16)
+public class Java16Test extends FATServletClient {
+
+    private static final String APP_NAME = "java16-app";
+
+    @Server("java11_fat-java16-server")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        // NOTE: This FAT uses a pre-compiled application which is compiled at the bytecode
+        // level of JDK 16, which is higher than what our build systems normally use
+        // Source for the app can be found here: https://github.com/gjwatts/liberty-misc/tree/java16-app/src/main/java
+        server.addInstalledAppForValidation(APP_NAME);
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+//        server.stopServer("CWWJP9991W"); // warnings for JPA's auto drop/create tables
+        server.stopServer();
+    }
+
+    @Test
+    public void testJava16App() throws Exception {
+        String appResponse = HttpUtils.getHttpResponseAsString(server, APP_NAME + '/');
+        assertContains(appResponse, "<<< EXIT SUCCESSFUL");
```

Comment:
If this assertion fails and you are getting a 404 at this endpoint likely the issue is that an exception was thrown. See comments on app on the best way to debug this.